ksqldb-rocksdb-config-setter/pom.xml

```diff
@@ -35,6 +35,12 @@
             <artifactId>kafka-streams</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.rocksdb</groupId>
+            <artifactId>rocksdbjni</artifactId>
+            <version>5.18.4</version>
```

Comment:
nit:  we could put this in the properties section in the ksql parent pom

Reply:
Normally I'd agree but I don't feel strongly in this case since ksqlDB shouldn't be in the business of interacting with RocksDB directly (we should be going through Streams APIs for that) which means this dependency is specific to this config setter module. 

OTOH, it's very little effort to move it and maybe we'll add more special modules later. I'll move it.